pkgs/data/themes/catppuccin-gtk/create-theme.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/scripts/create_theme.py b/scripts/create_theme.py
+index 074dc43..7e7a8eb 100644
+--- a/scripts/create_theme.py
++++ b/scripts/create_theme.py
+@@ -10,7 +10,7 @@ from scripts.utils import replacetext, zip_multiple_folders
+ from scripts.var import def_color_map, repo_dir, src_dir, theme_name, work_dir
+ 
+ 
+-def create_theme(types: List[str], accents: List[str], dest: str, link: bool = False, 
++def create_theme(types: List[str], accents: List[str], dest: str, link: bool = False,
+     name: str = theme_name, size: str = "standard", tweaks=[], zip = False, recreate_assets = False) -> None:
+ 
+     try:
+@@ -34,7 +34,11 @@ def create_theme(types: List[str], accents: List[str], dest: str, link: bool = F
+             os.chdir(work_dir)
+             subprocess.call("./build.sh", shell=True) # Rebuild all scss
+             subprocess.call(install_cmd, shell=True) # Install the theme globally for you
+-            subprocess.call("git reset --hard HEAD", shell=True)  # reset colloid repo to original state
++            # subprocess.call("git reset --hard HEAD", shell=True)  # reset colloid repo to original state
```

Comment:
This part is purely nix-specific, I suppose?

Reply:
That's correct the build scripts assumes colloid is a submodule but this derivation uses `colloid-gtk-theme` as an input.